src/components/PersistentDrawer.js

```diff
@@ -308,15 +296,15 @@ class PersistentDrawer extends React.Component {
                 onClick={this.handleDrawerOpen}
                 className={classNames(classes.menuButton, open && classes.hide)}
               >
-                <GearIcon />
+                <GearIcon className={classes.gearIcon} />
               </IconButton>
               <Typography
                 variant="title"
                 color="inherit"
-                className={styles.flex}
+                className={classes.headerText}
                 noWrap
               >
-                Coconut Interpreter
+                COCONUT EDITOR
```

Comment:
Change back to Capital Case with the css change above

Reply:
Done